### DISCUSSION TITLE
"LaserGRBL" software imports vector graphics

### DISCUSSION THREAD

**Discussion by lianzaozi** [arkypita/LaserGRBL#1534] [Q&A]
Can the "LaserGRBL" software add the function of importing vector graphics (svg dxf plt, etc.)? These files have the advantage that they have layer/color information, can identify layers or colors, and define different layers or colors Different laser power and speed.

**Comment by StuartB4** [comment]
It does do .svg files.

**Comment by lianzaozi** [comment]
that's good.

My question now is how to fill the shape in the svg file in the software?

**Comment by arkypita** [comment]
you can use your favorite vector graphics program to create an infinite number of fills through the tools that these programs usually offer: patterns, strokes, clipping ...

https://graphicdesign.stackexchange.com/questions/45577/create-hatching-in-inkscape
